examples/sample_identity_map_client.py

```diff
@@ -3,11 +3,11 @@
 from uid2_client import IdentityMapClient, IdentityMapInput
 
 
-# this sample client takes email addresses or phone numbers as input and generates an IdentityMapResponse object
-# which contains raw uid or the reason why it is unmapped
+# this sample client takes email addresses or phone numbers or hashed versions of either as input and generates
+# an IdentityMapResponse object which contains raw uid or the reason why it is unmapped
```

Comment:
this comment doesn't match what's in _usage()

Reply:
I updated the comment